apps/oc_chef_authz/src/oc_chef_authz_db.erl

```diff
@@ -148,6 +148,19 @@ statements(pgsql) ->
       <<"UPDATE groups SET last_updated_by= $1, updated_at= $2, name= $3"
         "WHERE id= $4">>},
      {delete_group_by_id, <<"DELETE FROM groups WHERE id= $1">>},
+     {list_policies_for_org, <<"SELECT name, policy_group FROM policies WHERE org_id= $1">>},
```

Comment:
Is order important for the returned records? (Would it be more useful if we ordered by policy_group, name?) 

Secondarily - would it make sense to have policy group as its own table? Or is group more like a tag/attribute of the policy?